src/wsl_tools/__init__.py

```diff
@@ -63,6 +62,22 @@ def from_dotdesktop(cls, app_def: Path) -> Optional[WSLApp]:
         icon = de.getIcon()
         if name:
             return cls(name, generic_name, cmd, gui, icon)
+        else:
+            # This is a symlink
+            linux_path = str(app_def)[str(app_def).index(r"\\wsl$") + 7:]
+            linux_path = linux_path[linux_path.index("\\"):].replace("\\", "/")
+            # Turn symlink target into Windows Path
+            symlink = distro.get_cmd_output(f"readlink -f -e '{linux_path}'")
```

Comment:
No need to call readlink, unc_path_from_cmd already uses realpath to follow symlinks